clang/test/Modules/no-external-type-id.cppm

```diff
@@ -23,7 +23,7 @@ export module b;
 import a;
 export int b();
 
-// CHECK: <DECL_FUNCTION {{.*}} op8=4048
```

Comment:
@ChuanqiXu9 - FYI, I am updating the test baseline. The `op8` parameter is type ID and it is based on the number of builtin types. When a new type is added the test fails because the expected type ID value does not match anymore. Please consider updating the test baseline to be resistant against this. :)